src/views/compute/DeployVM.vue

```diff
@@ -275,6 +275,16 @@
                   </div>
                 </template>
               </a-step>
+              <a-step
+                :title="this.$t('enter BIOS setup')"
+                :status="zoneSelected ? 'process' : 'wait'">
+                <template slot="description">
+                  <div v-if="zoneSelected">
+                    <a-checkbox
+                      @change="enterBiosChoiceChanged">enter BIOS setup</a-checkbox>
```

Comment:
if it's just a setter, we can also write shorthand like this:
```
@change="val => bootIntoBios = val"
```

Reply:
i'll test this, i didn't get any other way to work, yet.